modules/lightning_features/lightning_media/src/MediaHelper.php

```diff
@@ -62,7 +62,7 @@ public function getFileExtensions($check_access = FALSE, array $target_bundles =
         // If the field is a FileItem or any of its descendants, we can consider
         // it a file field. This will automatically include things like image
         // fields, which extend file fields.
-        if (is_a($field->getItemDefinition()->getClass(), FileItem::class, TRUE)) {
+        if (!empty($field) && is_a($field->getItemDefinition()->getClass(), FileItem::class, TRUE)) {
```

Comment:
@phenaproxima , @balsama -

It is possible under the following scenario in which the `$field` _could_ be empty and therefore would result in a fatal error due to treating `$field` as always being an object.

*Scenario:*

1. An administrator (or a user with permissions) _changes_ the field definition of a Media bundle by deleting the source field. For example, on the Image media bundle - the "image" field which is set as the source is deleted.
2. An administrator (or a user with permissions) adds a _new_ image field (not that which was previously set and now removed).
3. A content creator (someone with content creation access for creating new media by uploading a new file on the form widget (e.g. Entity Browser & File Upload) during the creation of a new content.